menas/src/test/scala/za/co/absa/enceladus/menas/integration/controllers/SchemaApiIntegrationSuite.scala

```diff
@@ -282,6 +286,105 @@ class SchemaApiIntegrationSuite extends BaseRestApiTest {
     }
   }
 
+  s"POST $apiUrl/upload" should {
+    "return 201" when {
+      "a copybook has no errors" should {
+        "return a new version of the schema" in {
+          val schema = SchemaFactory.getDummySchema()
+          val responseCreated = sendPost[Schema, Schema](s"$apiUrl/create", bodyOpt = Some(schema))
```

Comment:
How about we use `SchemaFixtureService.add(schema)` to keep the test cases simple?